plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/repository/LibraryClasspathContainerResolverService.java

```diff
@@ -73,7 +74,7 @@
   @Override
   public IStatus resolveAll(IJavaProject javaProject, IProgressMonitor monitor) {
     try {
-      IStatus status = Status.OK_STATUS;
+      MultiStatus status = StatusUtil.multi(this, Messages.getString("TaskResolveLibrariesError"));
```

Comment:
Why start with an error?

Reply:
It's a `MultiStatus` so it actually starts with OK.  It becomes not-OK only when a non-OK child is added.